### PR TITLE
weatherman reports only 0.5 inch of foolish white stuff

### PR DESCRIPTION
**Pull request by dudethatbe**
https://weatherishappening.network/@WEATHERISHAPPENING/111715704504092523

### CODE REVIEW

File: app/components/weather-selector.js [hunk 1]
```diff
@@ -2,8 +2,6 @@ import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
 
 export default class WeatherSelector extends Component {
-  @tracked currentWeather = "snowy";
```
Comment (dudethatbe) [inline app/components/weather-selector.js:5]: seems like this wasn't necessary so I deleted it

Comment (mcfiredrill) [reply app/components/weather-selector.js:5]: isnt' that used in the template?

Comment (dudethatbe) [reply app/components/weather-selector.js:5]: uh yeah woops
can the template reference the current weather stored in the service instead?

Comment (mcfiredrill) [reply app/components/weather-selector.js:5]: yeah you could inject the service and do it that way instead

**Comment by dudethatbe** [comment]
i removed the `text-2xl` class from the dropdown and looks better now

**Comment by mcfiredrill** [comment]
LGTM

**Pull request merged**